templates/security/CMakeLists.txt.forInstall

```diff
@@ -33,19 +33,19 @@ target_include_directories(securityImpl
 include_directories(${NATIVECLIENT_INCLUDE_DIR})
```

Comment:
This whole file should probably be replaced with the FindGeodeNative.cmake.

Reply:
Different story for that perhaps?  Doesn't seem as related to initial story.